### PR TITLE
Fix home component redirect and adapt reports view layout

### PR DESCRIPTION
**Pull request by thebeyondr**
This pull request includes the following changes:

- Updated the Home component to redirect to "/reports"

- Refactored the contributionAmounts calculation in VoicedeckStats

- Adapted the reports view layout on wide screens using grid display

These changes fix the issue with the Home component redirect and improve the layout of the reports view.

### CODE REVIEW

**Comment by baumstern** [comment]
redirect is still not wokring

File: app/profile/[address]/settings/page.tsx [hunk 1]
```diff
@@ -1,14 +1,29 @@
...
+				>
+					<ChevronLeft
+						size={24}
+						className="text-vd-blue-400 group-hover:-translate-x-2 transition-transform duration-300 ease-in-out"
+					/>
+					<p className="font-semibold text-sm uppercase text-vd-blue-500 tracking-wider">
+						Profile
+					</p>
+				</Link>
+				<div className="p-2" />
```
Comment (0xRowdy) [inline app/profile/[address]/settings/page.tsx:26]: 👀

**Review by baumstern — APPROVED** [review]
redirect works!

File: components/settings/user-info.tsx [hunk 2]
```diff
@@ -27,44 +25,33 @@ const UserInfo = () => {
...
+						<AlertDescription className="truncate">
+							{ensName ? ensName : "No ENS name found"}
+						</AlertDescription>
+					</Alert>
+					<small className="text-sm">
+						We'll use your ENS name if you have one.
 					</small>
 				</div>
-				<h2 className="text-xl md:text-2xl font-semibold md:py-6">Account</h2>
+				<div className="py-4" />
```
Comment (0xRowdy) [inline components/settings/user-info.tsx:44]: 👀

**Review by 0xRowdy — APPROVED** [review]
lgtm

**Pull request merged**